apps/src/componentLibrary/checkbox/checkbox.module.scss

```diff
@@ -16,6 +16,8 @@
 
   span {
     margin-left: 8px;
+    user-select: none;
+    -webkit-user-select: none;
```

Comment:
for the CSS newbies among us (me), can you explain what this does?

Reply:
I had the same question, thank you for asking Bethany!

Reply:
The user-select CSS property controls whether the user can select text. This doesn't have any effect on content loaded as part of a browser's user interface (https://developer.mozilla.org/en-US/docs/Web/CSS/user-select)
none - means you can not select the text

Reply:
That'll only affect checkbox labels. And it's needed to prevent users from accidental selecting the text when double clicking the checkbox
Initially it was selectable, but @breville raised this question in one of the PRs and it was quite reasonable

Reply:
ok, thanks for the context

Reply:
But actually yeah, I was also hesitated here. I guess that user-select:none should only be applicable to dropdown/checkbox-dropdown, so we might leave regular checkboxes selectable.
@rshipp thanks for asking this question!
@breville what do you think?

Reply:
I do believe in the specific case we discussed, it would be worth having this attribute to avoid text selection when someone rapidly toggles an option.  But if we can think of cases in which users might want to select text, we could leave this attribute to be used in specific cases, rather than in the core component.

Reply:
Agreed, I'll remove these 2 lines of code and make checkbox label text selectable